UPGRADE-GUIDE.md

```diff
@@ -3,6 +3,9 @@ id: Upgrade guide
 section: developer-resources
 ---
 
+Note: this guide details upgrading from PatternFly V3 to V4, and references [major release 2020.07](/v4/developer-resources/release-notes/html#2020.07-release-notes-2020-06-05) from 2020-06-05.
```

Comment:
* FWIW in our docs, we'll bold "Note:". Might be worth it here, too? 
* we use `v3/4` (lowercase) everywhere I've seen
* took the `/v4` out - do you know if that works? When we link to a component, we just link to `/components/...`
* looks like there is a missing `/` in the link compared to the URL I get if I click the anchor on org 

```suggestion
**Note:** this guide details upgrading from PatternFly v3 to v4, and references [major release 2020.07](/developer-resources/release-notes/html/#2020.07-release-notes-2020-06-05) from 2020-06-05.
```